README.md

```diff
@@ -14,6 +14,7 @@ For more information, [visit the home page for HoRNDIS on my site](http://www.jo
 ### From Homebrew
 
 ```sh
+sudo mount -uw /
```

Comment:
Why is this necessary?  That should only be necessary in single user mode, right?

Reply:
without this step, Catalina blocks the installation.